lib/galaxy/selenium/navigation.yml

```diff
@@ -71,7 +71,10 @@ preferences:
     sign_out: "#edit-preferences-sign-out"
     change_password: "#edit-preferences-password"
     manage_information: '#edit-preferences-information'
+    manage_api_key: '#edit-preferences-api-key'
     current_email: "#user-preferences-current-email"
+    api_key: "[data-label='Current API key:'] > input"
```

Comment:
It looks like you defined this but didn't use it in the code? Can you do that? Otherwise this looks awesome - thanks @davebx !

Reply:
It wasn't usable in the code, but I left it there with the reasoning that it might become usable one day. Then again, at that point it might be a different selector entirely, so it does make sense to remove it.